Do not use SetIdle, it is deprecated

On canary build, there is warning below, it needs to be fixed.

```
> @google-cloud/profiler@0.2.0 install /tmp/testprof/node_modules/@google-cloud/profiler
> node-gyp rebuild

make: Entering directory '/tmp/testprof/node_modules/@google-cloud/profiler/build'
  CXX(target) Release/obj.target/sampling_heap_profiler/bindings/sampling-heap-profiler.o
  SOLINK_MODULE(target) Release/obj.target/sampling_heap_profiler.node
  COPY Release/sampling_heap_profiler.node
  CXX(target) Release/obj.target/time_profiler/bindings/time-profiler.o
../bindings/time-profiler.cc: In function ‘Nan::NAN_METHOD_RETURN_TYPE SetIdle(Nan::NAN_METHOD_ARGS_TYPE)’:
../bindings/time-profiler.cc:92:31: warning: ‘void v8::CpuProfiler::SetIdle(bool)’ is deprecated: Use Isolate::SetIdle(bool) instead. [-Wdeprecated-declarations]
   cpuProfiler->SetIdle(is_idle);
                               ^
In file included from /usr/local/google/home/aalexand/.nvm/versions/node/v11.0.0-v8-canary201807140f69779e03/include/node/v8.h:26:0,
                 from /usr/local/google/home/aalexand/.nvm/versions/node/v11.0.0-v8-canary201807140f69779e03/include/node/v8-profiler.h:10,
                 from ../bindings/time-profiler.cc:17:
/usr/local/google/home/aalexand/.nvm/versions/node/v11.0.0-v8-canary201807140f69779e03/include/node/v8-profiler.h:360:22: note: declared here
                 void SetIdle(bool is_idle));
                      ^
/usr/local/google/home/aalexand/.nvm/versions/node/v11.0.0-v8-canary201807140f69779e03/include/node/v8config.h:324:3: note: in definition of macro ‘V8_DEPRECATED’
   declarator __attribute__((deprecated(message)))
   ^~~~~~~~~~
  SOLINK_MODULE(target) Release/obj.target/time_profiler.node
  COPY Release/time_profiler.node
make: Leaving directory '/tmp/testprof/node_modules/@google-cloud/profiler/build'
```

--- a/bindings/time-profiler.cc
+++ b/bindings/time-profiler.cc
@@ -87,11 +87,6 @@ NAN_METHOD(SetSamplingInterval) {
   cpuProfiler->SetSamplingInterval(us);
 }
 
-NAN_METHOD(SetIdle) {
-  bool is_idle = info[0].As<Boolean>()->BooleanValue();
-  cpuProfiler->SetIdle(is_idle);
-}
-
 NAN_MODULE_INIT(InitAll) {
   Nan::Set(target, Nan::New("startProfiling").ToLocalChecked(),
     Nan::GetFunction(Nan::New<FunctionTemplate>(StartProfiling)).ToLocalChecked());
@@ -99,8 +94,6 @@ NAN_MODULE_INIT(InitAll) {
     Nan::GetFunction(Nan::New<FunctionTemplate>(StopProfiling)).ToLocalChecked());
   Nan::Set(target, Nan::New("setSamplingInterval").ToLocalChecked(),
     Nan::GetFunction(Nan::New<FunctionTemplate>(SetSamplingInterval)).ToLocalChecked());
-  Nan::Set(target, Nan::New("setIdle").ToLocalChecked(),
-    Nan::GetFunction(Nan::New<FunctionTemplate>(SetIdle)).ToLocalChecked());
 }
 
 NODE_MODULE(time_profiler, InitAll);

